prow/config.yaml

```diff
@@ -7181,6 +7181,30 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
+  - name: pull-tf-minigo-cc
+    agent: kubernetes
+    always_run: true
+    skip_report: true
+    #context: pull-tf-minigo-cc
+    branches:
+    - master
+    rerun_command: "/test pull-tf-minigo-cc"
+    trigger: "/test pull-tf-minigo-cc"
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/tensor-go/cc-base:latest
+        command:
+        - "/bin/sh -c \"cp -r /app/cc/tensorflow ./cc/tensorflow && bazel test cc:all --compilation_mode=dbg --define=board_size=9\""
+        #- bazel test cc:all --compilation_mode=dbg --define=board_size=19
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "2Gi"
```

Comment:
How much cpu and memory should the job request?

Reply:
no idea?  :man_shrugging: `bazel build` will probably eat up all we give it.

Reply:
this is just a request, 2gi is a reasonable starting point for a bazel build.